frontend/workflows/projectSelector/src/project-group.tsx

```diff
@@ -224,7 +229,9 @@ const ProjectGroup: React.FC<ProjectGroupProps> = ({ title, group, displayToggle
               {!state?.loading && state?.projectData?.[key]?.linkGroups && (
                 <ProjectLinks
                   linkGroups={state?.projectData?.[key]?.linkGroups ?? []}
-                  name={[key]}
+                  currentKey={[key]}
+                  keyWithQLinksOpen={keyWithQLinksOpen}
+                  setKeyWithQLinksOpen={setKeyWithQLinksOpen}
```

Comment:
Do we need to do the logic for key comparison in the ProjectLinks component? Should we do that here instead of passing around strings and then pass in something like `open` and `onOpen` as a callback